packages/client/src/hooks/auth/useUser.js

```diff
@@ -2,12 +2,12 @@ import { useQuery } from '@tanstack/react-query';
 import { loginCheck, getUser } from '../../apis/authApi';
 
 export const useCheckLogin = () => {
- return useQuery(['isLoggedIn'], loginCheck);
-}
+  return useQuery(['isLoggedIn'], loginCheck);

```

Comment:
이 경우는 바로 arrow function + logic 하지 않아도 될 것 같아요

```suggestion
export const useCheckLogin = useQuery(['isLoggedIn'], loginCheck);
```